modules/DateTime.mjs

```diff
@@ -0,0 +1,9 @@
+import { DateTime } from "luxon";
+export default function updateDate() {
+  const dateContainer = document.querySelector("#date-container");
+  dateContainer.innerHTML = "";
+  const date = DateTime.now().toFormat("FF");
+  console.log(date);
+
+  dateContainer.append(date);
+}
```

Comment:
- [ ] please make use of ES6 arrow functions as mentioned in the requirements instead of the function keyword.
- [ ] the date and time are not visible in the application right now please make sure to add it before requesting another code review.